metaseq/dataclass/configs.py

```diff
@@ -560,6 +560,32 @@ class CheckpointConfig(MetaseqDataclass):
             "argparse_alias": "--cloud-dir",
         },
     )
+    cloud_eval_script_path: Optional[str] = field(
```

Comment:
We are calling these flags `cloud` but we are not using them for blob nor S3. Could we either make the logic generic or call them `nfs_` instead?

Reply:
Also, if we are adding new flags can we remove the `evaluate_` ones added previously? https://github.com/facebookresearch/metaseq/pull/411

Reply:
You are right. It is is also coupled to NFS, as we are checking for existence of files, and the eval script won't work on non NFS files. 
- will change it to `nfs_`.
- remove evaluate flag